lms/djangoapps/instructor_task/api_helper.py

```diff
@@ -87,6 +87,7 @@ def _get_xmodule_instance_args(request, task_id):
     The `task_id` is also passed to the tracking log function.
     """
     request_info = {'username': request.user.username,
+                    'id': request.user.id,
```

Comment:
rename to `user_id` instead of just `id`.  Otherwise, it's unclear what the `id` is for - one would assume it's the `id` of the request?